client/js/bundles/faq/faqlist.js

```diff
@@ -12,13 +12,17 @@
  */
 
 import RegisterFlyout from '@DpJs/components/user/RegisterFlyout'
+import DpFaqSupport from '@DpJs/components/faq/DpFaqSupport'
```

Comment:
```suggestion
import DpFaqSupport from '@DpJs/components/faq/DpFaqSupport'
import RegisterFlyout from '@DpJs/components/user/RegisterFlyout'
```

Reply:
The Linter should complain about that too. If it didn't, please tell me, then we can have a look into your settings